infra/drc/nginx.yml

```diff
@@ -0,0 +1,95 @@
+---
+
+# Deployment
+
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: drc-nginx
+  namespace: gemma
```

Comment:
Was de namespace niet vervangen met "zgw"?

Reply:
in nieuwe cluster wel, in oude niet